truth_justice_cake_cutting_demo.py

```diff
@@ -0,0 +1,43 @@
+"""
+
+Programmer: Ofec Israel
+
+"""
+
+
```

Comment:
See in the other demo files, how to initialize the logger inside the demo program.